spec/helpers/admin/users_helper_spec.rb

```diff
@@ -1,6 +1,33 @@
 require "rails_helper"
 
 describe Admin::UsersHelper do
+  describe "#role_options" do
+    let(:user) { create(:user) }
+
+    it "returns base roles" do
```

Comment:
Quick, non-blocking nit-pick: could we please aggregate the failures here (and in the tests below) since we have multiple expectations within this single block?
```suggestion
    it "returns base roles", :aggregate_failures do
```